classes/cinema_ticket_processor_class.rb

```diff
@@ -0,0 +1,24 @@
+class CinemaTicketsProcessor
+  attr_accessor :movie_name, :ticket_cost, :screening_time
+
+  def initialize(movie_name, ticket_cost, screening_time)
+    @movie_name = movie_name
+    @ticket_cost = ticket_cost
+    @screening_time = screening_time
+  end
+
+  def ticket_printer
+    puts @movie_name
+    puts @screening_time
+    puts @ticket_cost
```

Comment:
Would it work without `@`? And why?

Reply:
It works, I guess it is because I'm using those variables inside the class, is that right?